helm/examples/metrics/README.md

```diff
@@ -20,3 +20,37 @@ For example, the following adds the Prometheus configuration as the collection s
 ```code
 helm install mycluster ./hpcc -f <path>/prometheus_metrics.yml
 ```
+### Prometheus Collection System
+When enabled and properly configured, the HPCC metrics for Prometheus feature exposes an HTTP service on each pod of any HPCC component
+which reports metrics. It also annotates all HPCC pods to be discoverable by Prometheus servers and provides the connectivity inforation required, such as
+the port to liston on, and the url path on which to serve Prometheus formatted metrics. Autodiscovery annotations can be disabled via metrics.sinks[type=prometheus].settings.autodiscovery
+
+####Prometheus Metric Service Discovery
+On the Prometheus server side, an "additionalScrapeConfigs" entry is necessary to link the HPCC metrics services Prometheus
+
+For example, the following can be added to Prometheus:
+
+```code
+    additionalScrapeConfigs:
+    - job_name: 'hpcc-per-pod-metrics'
+      scrape_interval: 5s
+      metrics_path: /metrics
+      kubernetes_sd_configs:
+        - role: pod
+      relabel_configs:
+      - source_labels: [ __meta_kubernetes_pod_annotation_prometheus_io_scrape ]
+        action: keep
+        regex: true
+      - source_labels: [ __meta_kubernetes_pod_annotation_prometheus_io_path ]
+        action: replace
+        target_label: __metrics_path__
+        regex: (.+)
+      - source_labels: [ __address__, __meta_kubernetes_pod_annotation_prometheus_io_port ]
```

Comment:
Doesn't seem to apply when __address__ (pre-label) doesn't contain a port. Port seems to be drawn from the pod's containerpod.